### ISSUE TITLE
Validation error when using additionalProperties object

### ISSUE THREAD

**Issue by badosa** [bugventure/jsen#56]
jsen incorrectly returns _true_ (valid) when the following schema 

```
{
  "$schema": "http://json-schema.org/draft-04/schema#",

  "type": "object",
  "additionalProperties": {
    "type": "object",
    "properties": {
      "label": {
        "type": "string"
      }
    },
    "additionalProperties": false
  }
}
```

is used to validate the following JSON

```
{
  "one" : {
    "label" : "1"
  },
  "two" : {
    "number" : "2"
  }
}
```

jsen seems to be taking into account only the first property ("one") as it returns _false_ (not valid) for  

```
{
  "one" : {
    "number" : "1"
  },
  "two" : {
    "label" : "2"
  }
}
```

**Comment by DavidGrinberg** [comment]
I'm getting this as well. It seems like in the javascript that jsen is generating for the validator there is an overflow on one of the counters.

**Comment by bugventure** [comment]
Thanks for reporting! Fixed for v0.6.2.
